.github/workflows/benchmarks.yml

```diff
@@ -38,20 +38,10 @@ jobs:
         uses: google-github-actions/setup-gcloud@d51b5346f85640ec2aa2fa057354d2b82c2fcbce #v1.0.1
       - name: microbenchmarks
         uses: martinbonnin/run-benchmarks@ef9043b9a646a109f7381a4bf20f82ead5cbd382 #main
```

Comment:
Need to call `run-benchmarks.main.kts` instead?

Reply:
Woops, right!